### DISCUSSION TITLE
阿里云盘报错

### DISCUSSION THREAD

**Discussion by Guan612** [AlistGo/alist#806] [Q&A] [answered]
Post "https://auth.aliyundrive.com/v2/account/token": dial tcp: lookup auth.aliyundrive.com on 192.168.2.50:53: server misbehaving
显示这个错误

**Comment by xhofe** [comment, marked as answer]
dns。。。

**Reply by Guan612** [reply]
好的了解了，谢谢

**Reply by xx668888** [reply]
怎么解决的呢

**Reply by wenxuan70** [reply]
修改dns服务器地址。
Linux环境下修改/etc/resolv.conf文件。

**Reply by xx668888** [reply]
> 修改dns服务器地址。 Linux环境下修改/etc/resolv.conf文件。

我是docker 腾讯云服务器，后来重启了一下docker 然后可以访问了

**Reply by AndrewGuo128** [reply]
请问在/etc/resolv.conf内要如何配置啊，新手还不懂这个

**Reply by AndrewGuo128** [reply]
请问在/etc/resolv.conf内要如何配置啊？楼主解决了吗

**Reply by wenxuan70** [reply]
改下/etc/resolve.conf中的nameserver看下行不行，修改方法自己搜一下

**Comment by lzwmt** [comment]
要改成什么，换了好几个都不行

**Comment by wenxuan70** [comment]
国内常见公共dns都行，阿里的223.5.5.5，腾讯的119.29.29.29
